iaso/models/org_unit.py

```diff
@@ -132,7 +135,7 @@ class OrgUnitType(models.Model):
     objects = OrgUnitTypeManager()
 
     def __str__(self):
-        return "%s" % self.name
+        return f"#{self.pk} {self.name}"
```

Comment:
I don't have major objections to these changes, but I'm wondering why you feel it's a better way to format strings

Reply:
This format was useful in the admin UI to identify the correct objects.

I'm also relying on this format here https://github.com/BLSQ/iaso/blob/7c294c02e01152969bebe07024a54ae4e28a80fd/iaso/models/data_source.py#L313 but now that I think about it, I think it's too brittle… I'm gonna fix this…

Reply:
Fixed in https://github.com/BLSQ/iaso/pull/1836/commits/92379fc661356a01901c198b4da16b123a24ff9b